tests/network/bond/test_bond_modes.py

```
@@ -183,12 +198,18 @@ def vm_with_fail_over_mac_bond(
 
 
 @pytest.fixture()
-def bond_resource(index_number, nodes_available_nics, worker_node1):
+def bond_resource(index_number, hosts_common_available_ports, worker_node1):
     bond_idx = next(index_number)
+    if len(hosts_common_available_ports) < 2:
+        LOGGER.error(
+            f"Insufficient common NICs across worker nodes: found {len(hosts_common_available_ports)} "
+            f"({hosts_common_available_ports})"
+        )
+        raise AssertionError("At least 2 common NICs are required.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)